src/pages/en/guides/styling.mdx

```diff
@@ -481,6 +481,13 @@ Vue in Astro supports the same methods as `vue-loader` does:
 
 Svelte in Astro also works exactly as expected: [Svelte Styling Docs][svelte-style].
 
+## Markdown Styling
+
+Any Astro styling methods are available to a [Markdown layout component](/en/core-concepts/layouts/#markdownmdx-layouts).
+
+You can apply global styles to your Markdown content by adding [`<style>` tags](#global-styles) and [imported stylesheets](#external-styles) to the layout that wraps your page content. You can also add [CSS integrations](#css-integrations) including [Tailwind](/en/guides/integrations-guide/tailwind/).
```

Comment:
Do we need to say all these things again if they're already on this page and we're already saying "you can do anything on this page"?

Reply:
So, I *get* that... but I struggled. This is our canonical, "How to style Markdown" and it seems like the answer is.... don't? :sweat_smile: Style in the layout component instead?

I'm also accounting for people who search and end up down here at the bottom of the page, so wanted to call out the things from above that are useful to them.

Reply:
Ok, so sounds like the answer is "Yes, we do need to say these things"

I'm cool with that! 👍